examples/cluster-e2e/logging.tf

```diff
@@ -0,0 +1,28 @@
+#####################################################
+# LOGDNA Configuration
+# Copyright 2021 IBM
+#####################################################
+
+module "logdna_instance" {
+  source = "terraform-ibm-modules/observability/ibm//modules/logging-logdna"
```

Comment:
still source is reffering from registry

Reply:
To provision a logdna instance we need to source to registry level